src/main/java/sys/Parser.java

```diff
@@ -0,0 +1,54 @@
+package sys;
+
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.ExitCommand;
+import command.FindCommand;
+import command.ListCommand;
+import command.MarkCommand;
+import command.OccursCommand;
+import command.UnmarkCommand;
+
+/**
+ * Represents a parser used to decode user input.
+ */
+public class Parser {
+
+    /**
+     * Constructor for Parser.
+     */
+    Parser() {};
```

Comment:
No need to include this if you don't need a constructor: Java will have a default constructor even if you don't include it.